core/song_metadata_db.py

```
@@ -0,0 +1,397 @@
+        stmt = select(Song).where(
+            or_(
+                Song.song_id.like(keyword_pattern),
+                Song.filename.like(keyword_pattern),
+                Song.song_title.like(keyword_pattern),
+                Song.artist_name.like(keyword_pattern),
+                Song.source_dir.like(keyword_pattern),
+            )
+        )
+
+        if exclude_from_search:
+            stmt = stmt.where(Song.excluded_from_search == False)
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `False`; use `not Song.excluded_from_search:` for false checks

Replace with `not Song.excluded_from_search`

(E712)